### PR TITLE
[dashboard] don't show connection status alert in iframe

### PR DESCRIPTION
**Pull request by mustard-mh**
## Description

See [internal chat](https://gitpod.slack.com/archives/C01KGM9BH54/p1677142029929329?thread_ts=1676985970.685469&cid=C01KGM9BH54)

## Related Issue(s)

Fixes #

## How to test

No need

## Release Notes

```release-note
NONE
```

## Build Options:

- [ ] /werft with-github-actions
      Experimental feature to run the build with GitHub Actions (and not in Werft).
- [ ] leeway-no-cache
      leeway-target=components:all
- [ ] /werft no-test
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

- [ ] /werft publish-to-npm
- [ ] /werft publish-to-jb-marketplace
</details>

<details>
<summary>Installer Options</summary>

- [ ] with-ee-license
- [ ] with-slow-database
- [ ] with-dedicated-emulation
- [ ] with-ws-manager-mk2
- [ ] workspace-feature-flags
  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-gce-vm
      If enabled this will create the environment on GCE infra
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

**Review by gtsiolis — APPROVED** [review]
Approving to unblock merging, but holding in case we need someone from the `@gitpod-io/engineering-webapp` to take a closer look at the code changes.

/hold

**Review by geropl — APPROVED** [review]
LGTM

**Pull request merged**
